sql/core/src/test/scala/org/apache/spark/sql/execution/debug/DebuggingSuite.scala

```diff
@@ -24,14 +24,13 @@ import org.apache.spark.sql.catalyst.InternalRow
 import org.apache.spark.sql.catalyst.expressions.Attribute
 import org.apache.spark.sql.catalyst.expressions.codegen.CodegenContext
 import org.apache.spark.sql.execution.{CodegenSupport, LeafExecNode, WholeStageCodegenExec}
-import org.apache.spark.sql.execution.adaptive.DisableAdaptiveExecutionSuite
+import org.apache.spark.sql.execution.adaptive.{DisableAdaptiveExecutionSuite, EnableAdaptiveExecutionSuite}
 import org.apache.spark.sql.functions._
 import org.apache.spark.sql.test.SharedSparkSession
 import org.apache.spark.sql.test.SQLTestData.TestData
 import org.apache.spark.sql.types.StructType
 
-// Disable AQE because the WholeStageCodegenExec is added when running QueryStageExec
```

Comment:
Instead of removal, shall we recover this for `DebuggingSuite` at line 100?

Reply:
@dongjoon-hyun - sorry can you elaborate more by recovering? Here I kept the AQE-runnable unit tests under `DebuggingSuiteBase`, and have a non-AQE `DebuggingSuite` and AQE `DebuggingSuiteAE`.

Reply:
oh do you mean move the comment to line 100?

Reply:
@dongjoon-hyun - updated to move the comment to line 100, let me know if my understanding is not correct, thanks.

Reply:
Thanks!